src/component/plotly-graph/render.js

```diff
@@ -51,7 +51,7 @@ function render (info, opts, sendToMain) {
   }
 
   const imgOpts = {
-    format: (PRINT_TO_PDF || PRINT_TO_EMF) ? 'svg' : format,
+    format: (PRINT_TO_PDF || PRINT_TO_EMF) ? 'svg' : (format === 'json' ? 'full-json' : format),
```

Comment:
This is getting a bit tortured... I would recommend hoisting this out into a little sequence of `ifs` or a `switch` or something :)

Reply:
Done in dbe395c3c5293725a3e72bca838fe9917bb91e82